Editor/SettingsAnalysis/PlayerSettingsAnalyzer.cs

```diff
@@ -84,9 +89,14 @@ internal static bool IsSplashScreenEnabledAndCanBeDisabled()
             return (bool)licenseAllowsDisablingProperty.GetValue(null, null);
         }
 
-        internal static bool IsNotSpeakerModeMono()
+        internal static bool IsSpeakerModeMono()
+        {
+            return AudioSettings.GetConfiguration().speakerMode == AudioSpeakerMode.Mono;
+        }
+
+        internal static void FixSpeakerMode()
         {
-            return AudioSettings.speakerMode != AudioSpeakerMode.Mono;
+            AudioSettings.speakerMode = AudioSpeakerMode.Mono;
```

Comment:
A small detail:

There is an API warning when running this on Unity 2021.

We could check with Marco if (just like the unit tests) we should call `AudioSettings.Reset(AudioSpeakerMode.Mono);`

That is a cleaner way since at least version 2019.3 to use the API (so no more warnings if we decide to not support lower versions).

Reply:
I wasn't aware `AudioSettings.speakerMode` was deprecated. Yes, we should definitely use `Reset`.